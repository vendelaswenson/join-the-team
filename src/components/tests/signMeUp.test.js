import SignMeUp from '../signMeUp'
import { render, cleanup } from '@testing-library/react'
import React from 'react'
import { MemoryRouter } from 'react-router-dom'

afterEach(() => {
  cleanup()
})

describe('Component', () => {
  it('renders without crashing', async () => {
    const utils = render(<SignMeUp />, { wrapper: MemoryRouter })
    expect(utils).toMatchSnapshot()
    expect(utils).toBeTruthy()
  })
})
