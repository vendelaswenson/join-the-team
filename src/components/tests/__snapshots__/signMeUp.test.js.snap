// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component renders without crashing 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="flex vh-100 items-center"
      >
        <div
          class="bg-cogs white pv4 ph5 h-100"
        >
          <h1
            class="f1 flex flex-column"
          >
            <span>
              Join
            </span>
             
            <span>
              the
            </span>
             
            <span>
              team
            </span>
          </h1>
          <div>
            <h1>
              List of Names
            </h1>
            <ul />
          </div>
        </div>
        <form>
          <input
            name="firstName"
            placeholder="First name"
          />
          <select
            name="category"
          >
            <option
              value=""
            >
              Select...
            </option>
            <option
              value="A"
            >
              Male
            </option>
            <option
              value="B"
            >
              Female
            </option>
          </select>
          <textarea
            name="aboutYou"
            placeholder="About you"
          />
          <p />
          <input
            type="submit"
          />
        </form>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="flex vh-100 items-center"
    >
      <div
        class="bg-cogs white pv4 ph5 h-100"
      >
        <h1
          class="f1 flex flex-column"
        >
          <span>
            Join
          </span>
           
          <span>
            the
          </span>
           
          <span>
            team
          </span>
        </h1>
        <div>
          <h1>
            List of Names
          </h1>
          <ul />
        </div>
      </div>
      <form>
        <input
          name="firstName"
          placeholder="First name"
        />
        <select
          name="category"
        >
          <option
            value=""
          >
            Select...
          </option>
          <option
            value="A"
          >
            Male
          </option>
          <option
            value="B"
          >
            Female
          </option>
        </select>
        <textarea
          name="aboutYou"
          placeholder="About you"
        />
        <p />
        <input
          type="submit"
        />
      </form>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
